app/views/collections/ugc.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+module Collections
+  class Ugc < ApplicationView
+    include BrowsableView
+    include SearchableView
+
+    def js_vars
+      [
+        {
+          name: 'pageName', value: 'e7a_1418'
+        }
+      ]
+    end
+
+    def page_title
+      'Europeana - First World War'
+    end
+
+    def content
```

Comment:
Doesn't really matter as far as I'm concerned. The base_1418_url does however determine where the content of the iframe gets loaded in from. So from that point of view it pertains to the content.
I've removed the portal_url as there was a misunderstanding between myself and Andy as to what it's purpose was and it wasn't doing what Andy intended it to do in the first place.